packages/pds/src/account-manager/account-manager.ts

```diff
@@ -172,6 +172,10 @@ export class AccountManager {
     deactivated?: boolean
     refreshJwt?: string
   }) {
+    if (password && password.length > scrypt.NEW_PASSWORD_MAX_LENGTH) {
+      throw new InvalidRequestError('Password too long')
```

Comment:
How will these kinds of xrpc errors surface up to the oauth UI?  I wondered if that may rely on some different error class, or other explicit error handling.

Reply:
This is already handled here:

https://github.com/bluesky-social/atproto/blob/897ee3748bb40ae54970af3db5cb62ac479165de/packages/pds/src/account-manager/oauth-store.ts#L190-L196

Reply:
Also, the OAuth will already have validated that particular constraint through it's own Zod validation schema.